src/App.js

```diff
@@ -1,9 +1,34 @@
-function App () {
+import birdData from "./data/birds";
+import bonusItems from "./data/bonusItems";
+import "./App.css";
+import { useState } from "react";
+import Birdcards from "./components/Birdcards";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+
+function App() {
+
+  const [birds, setBirds] = useState([]);
+
+  function handleBirdCount(bird){
+    const birdObj = {...bird};
+    setBirds([...birds, birdObj])
+  }
+
   return (
-    <div>
-      <h1>Hello, world!</h1>
+    <div className="App">
+      <header>
+        <h1>Bird Sanctuary Lab</h1>
+      </header>
+      <main>
+        <div>
+          <Cart birds={birds} bonusItems={bonusItems} setBirds={setBirds}/>
+          <Checkout setBirds={setBirds}/>
+        </div>
+        <Birdcards birdData={birdData} handleBirdCount={handleBirdCount}/>
```

Comment:
Component names should start with a capital letter and then camel casing (BirdCards)